jupyterhub/singleuser.py

```diff
@@ -177,7 +181,7 @@ def start(self):
                 self._clear_cookie_cache,
                 self.cookie_cache_lifetime * 1e3,
             ).start()
-        super().start()
+        super(SingleUserNotebookApp, self).start()
```

Comment:
Can you add a py2 compat comment here, so we can more easily revert it when Python 2 support is dropped?